src/platforms/dart/migration.mdx

```diff
@@ -17,4 +17,5 @@ sidebar_order: 1000
     * `OperatingSystem` -> `SentryOperatingSystem`
     * `Orientation` -> `SentryOrientation`
     * `Request` -> `SentryRequest`
-    * `User` -> `SentryUser`
\ No newline at end of file
+    * `User` -> `SentryUser`
+    * `Orientation` -> `SentryOrientation`
```

Comment:
Orientation is no two times in this list.

Reply:
ups, missed it sorry